packages/app/components/constants.ts

```diff
@@ -69,3 +69,22 @@ export const firebaseConfig = {
   messagingSenderId: "835188230670",
   appId: "1:835188230670:web:1ee8ae3a9338c874de8a5f",
 };
+
+export const shimmer = (w: number, h: number) => `
```

Comment:
I think you could store this as an `svg` file, import it where it's used, and still pass in width/height props to the component since Next uses react svg

Reply:
ah I see it's used in a `blurDataUrl`. Still, the import resolves to a URL if this is its own file. and since our only use of this has width and height hardcoded, we could hardcode them in the file

Reply:
I'll pick this up later. I couldnt get it to work with my initial attempts.